src/overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -326,7 +321,7 @@ def from_dict(cls, obj_dict):
 
 class SoupState(ObjectState):
 
-    def __init__(self, position, ingredients=[], cooking_tick=-1, **kwargs):
+    def __init__(self, position, ingredients=[], cooking_tick=-1, cook_time=None, **kwargs):
```

Comment:
I wanted to add this cook_time before, but I believe there was a reason I was cautioned against it. @nathan-miller23 @micahcarroll maybe I should direct your attention to this?

Reply:
I remember @nathan-miller23 having a reason this (but don't remember exactly what) – we should probably wait for him to respond to this.